Mathlib/MeasureTheory/Measure/HasOuterApproxClosedProd.lean

```diff
@@ -9,22 +9,30 @@ import Mathlib.MeasureTheory.Measure.HasOuterApproxClosed
 /-!
 # Characterization of a finite measure by the integrals of products of bounded functions
 
-Given two Borel spaces `X` and `Y` satisfying `HasOuterApproxClosed`, a finite measure `μ`
-over `X × Y` is determined by the values `∫ z, f z.1 * g z.2 ∂μ`, for `f : X → ℝ` and `g : Y → ℝ`
-any bounded continuous functions.
+Given two finite families of Borel spaces `(i : ι) → X i` and `(j : κ) → Y j` satisfying
+`HasOuterApproxClosed`, a finite measure `μ` over `(Π i, X i) × (Π j, Y j)` is determined by
+the values `∫ p, (Π i, f i (p.1 i)) * (Π j, g j (p.2 j)) ∂μ`, for
+`f : (i : ι) → X i → ℝ` and `g : (j : κ) → Y j → ℝ`
+any families of bounded continuous functions.
 
-In particular, If `μ` and `ν` and two finite measures over `X` and `Y` respectively, then their
-product is the only finite measure `ξ` over `X × Y` such that for any two bounded continuous
-functions `f : X → ℝ` and `g : Y → ℝ` we have
-`∫ z, f z.1 * g z.2 ∂ξ = (∫ x, f x ∂μ) * (∫ y, g y ∂ν)`.
+In particular, If `μ` and `ν` and two finite measures over `Π i, X i` and `Π j, Y j` respectively,
+then their product is the only finite measure `ξ` over `(Π i, X i) × (Π j, Y j)`
+such that for any two families bounded continuous functions
+`f : (i : ι) → X i → ℝ` and `g : (j : κ) → Y j → ℝ` we have
+`∫ p, (Π i, f i (p.1 i)) * (Π j, g j (p.2 j)) ∂ξ =
+(∫ x, Π i, f i (x i) ∂μ) * (∫ y, Π j, g j (y j) ∂ν)`.
+
+We specialize these results to the cases where one of the families contains only one type.
 
 # Main statements
 
-* `ext_of_integral_mul_boundedContinuousFunction`: A finite measure on a product space is
+* `ext_of_integral_prod_mul_prod_boundedContinuousFunction`: A finite measure on a product space is
```

Comment:
Could you keep `ext_of_integral_mul_boundedContinuousFunction` in this list, and mention the difference between this one and `ext_of_integral_prod_mul_prod_boundedContinuousFunction`?